backend/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import 'extensions/mongoose.extension';
+import pathwayRouter from 'routes/pathway';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+// Fix Deprecation Warnings
+mongoose.set('useCreateIndex', true);
+mongoose.set('useUnifiedTopology', true);
+mongoose.set('useFindAndModify', false);
+
+const app = express();
+const PORT = 8000;
+const payloadSizeLimit = '50mb';
+
+app.use(bodyParser.json({ limit: payloadSizeLimit }));
+app.use(cors());
+
+mongoose.connect('mongodb://localhost/test', { useNewUrlParser: true });
```

Comment:
I think the name of the database can be changed without causing any issues

Reply:
I don't think it matters either so I'll change it to something better than `test`.  `pathways` perhaps?